app/views/admin/resources/new.html.haml

```diff
@@ -1 +1 @@
-= render 'form'
+= render 'form', resource: @resource
```

Comment:
There is an option [InstanceVariables](https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md#instancevariables) in haml_lint. Maybe we can disable this option?